strings/salesman_replacement.json

```diff
@@ -0,0 +1,49 @@
+{
+
+    "salesman": {
```

Comment:
GENERAL THOUGHTS on this replacement file:
these are replacing a lot of words you won't say normally, which is bad because if someone is trying to roleplay as the character you're referencing, they're already going to be replacing uncommon terminology. This should instead target commonly said phrases and let the roleplayer do the rest.

Reply:
Additionally, split this into "non-bracket" and "bracket" lists and have the matches in the bracket category get surrounded by brackets.

that way, when they say "big shot" it will stay as is, but when they say "hyperlink blocked" it will come out as "[hyperlink blocked]"

I wasn't joking around about doing the reference right if you're going to

Reply:
To be honest im really on the edge between it being a creative interpretation without the brackets so people who dont know the reference can still enjoy it, or if i should add in the brackets and be more like the source, even if people wont understand what things mean.

I guess brackets would be good honestly.